static/js/zamboni/reviewers.js

```diff
@@ -110,6 +112,11 @@ function initReviewActions() {
     // the value we're interested in.
     $data_toggle_hide.show();
     $data_toggle_hide.filter('[data-value~="' + value + '"]').hide();
+
+    $('.review-actions-attachment-reply').toggleClass(
```

Comment:
Just clarifying.. did you look for a way to add this to the template. Relying on JS to hide things can be tricky.

1. the element will be visible until js loads and this particular function is executed.
2. the element will still be in the dom, just not visible.

It would be much better to hide this element in the view (server side) OR to invert the logic and make it invisible by default and only make it "visible" on certain actions.

Reply:
All the toggling between action panels happens in this function above the line I added, including the other form inputs that need to be shown/hidden, so I don't think there's a way to avoid JS. I did realize I could just route the show/hide logic through that existing JS, though, so at least there isn't any _new_ JS.